backend/src/main/java/org/cryptomator/hub/api/VaultResource.java

```diff
@@ -218,32 +218,15 @@ private Response addAuthority(Vault vault, Authority authority, VaultAccess.Role
 	}
 
 	@DELETE
-	@Path("/{vaultId}/users/{userId}")
-	@RolesAllowed("user")
-	@VaultRole(VaultAccess.Role.OWNER) // may throw 403
-	@Transactional
-	@Produces(MediaType.APPLICATION_JSON)
-	@Operation(summary = "remove a member from this vault", description = "revokes the given user's access rights from this vault. If the given user is no member, the request is a no-op.")
-	@APIResponse(responseCode = "204", description = "member removed")
-	@APIResponse(responseCode = "403", description = "not a vault owner")
-	public Response removeUser(@PathParam("vaultId") UUID vaultId, @PathParam("userId") @ValidId String userId) {
-		return removeAuthority(vaultId, userId);
-	}
-
-	@DELETE
-	@Path("/{vaultId}/groups/{groupId}")
+	@Path("/{vaultId}/authority/{authorityId}")
 	@RolesAllowed("user")
 	@VaultRole(VaultAccess.Role.OWNER) // may throw 403
 	@Transactional
 	@Produces(MediaType.APPLICATION_JSON)
-	@Operation(summary = "remove a group from this vault", description = "revokes the given group's access rights from this vault. If the given group is no member, the request is a no-op.")
-	@APIResponse(responseCode = "204", description = "member removed")
+	@Operation(summary = "remove an authority from this vault", description = "revokes the given authority's access rights from this vault. If the given authority is no member, the request is a no-op.")
```

Comment:
```suggestion
	@Operation(summary = "remove a user or group from this vault", description = "revokes the given authority's access rights from this vault. If the given authority is no member, the request is a no-op.")
```

Reply:
Fixed in [b67cfdb](https://github.com/cryptomator/hub/pull/243/commits/b67cfdbb523156a8f6aced31e4997f3822e52a53)